static/app/components/modals/savedSearchModal/createSavedSearchModal.spec.tsx

```diff
@@ -47,6 +47,10 @@ describe('CreateSavedSearchModal', function () {
       method: 'POST',
       body: [],
     });
+    MockApiClient.addMockResponse({
```

Comment:
Ignore all of `MockApiClient.addMockResponse({...})` additions made in this PR - [this prevents CI from failing](https://develop.sentry.dev/development/testing/#api-fixtures)